cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -331,7 +453,14 @@ bool cros_gralloc_driver::get_resolved_format_and_use_flags(
 	struct combination *combo;
 
 	struct driver *drv = is_video_format(descriptor) ? drv_video_ : drv_render_;
-	if ((descriptor->use_flags & BO_USE_SCANOUT) && !(is_video_format(descriptor)))
+	if (!drv_ivshmem_) {
+		if (reload()) {
```

Comment:
reload will be always called if ACRN didn't expose ivshmem device to Android, then the overload is too heavy.
Can we just check the special width/height, then reload?